guidelines/index.html

```diff
@@ -2975,10 +2975,10 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
    
 </dd>
             
-            <dt class="new"><dfn id="dfn-perimeter" tabindex="0" aria-haspopup="dialog" data-dfn-type="dfn">perimeter</dfn></dt>
+            <dt class="new"><dfn id="dfn-perimeter" tabindex="0" aria-haspopup="dialog" data-dfn-type="dfn">外周 (perimeter)</dfn></dt>
 <dd class="new"><p class="change">[New]</p>	
-   <p>continuous line forming the boundary of a shape not including shared pixels, or the <a href="#dfn-bounding-boxes" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-bounding-boxes-2" title="[New]">minimum bounding box</a>, whichever is shortest.</p>
-  <aside class="example" id="example-16"><div class="marker">例</div><p>The perimeter calculation for a 2 CSS pixel perimeter around a rectangle is 4<em>h</em>+4<em>w</em>, where <em>h</em> is the height and <em>w</em> is the width. For a 2 CSS pixel perimeter around a circle it is 4𝜋<em>r</em>.</p></aside>
+   <p>共有されたピクセルがない形状の、境界を形成する連続線、又は<a href="#dfn-bounding-boxes" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-bounding-boxes-2" title="[New]">最小境界ボックス</a>、のいずれか短い方。</p>
```

Comment:
continuous line forming the boundary of a shape not including shared pixels というのは、こういうことでしょう。。
<https://stackoverflow.com/questions/69366664/shared-pixels-on-shared-edges>

continuous line：実線
<https://eow.alc.co.jp/search?q=continuous+line>

sharedの態は微妙ですが…以下はどうでしょう

```suggestion
   <p>共有するピクセルを含まない図形の境界を形成する実線、又は<a href="#dfn-bounding-boxes" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-bounding-boxes-2" title="[New]">最小境界ボックス</a>のいずれか短い方。</p>
```

Reply:
continuous line は実線、なるほどです！